scripts/gitlab/test-cpp.sh

```diff
@@ -11,7 +11,6 @@ mkdir -p $DIR
 cd $DIR
 cmake ..
 make VERBOSE=1 -j $THREADS
```

Comment:
Or alternatively use the `ninja` build system. Then you do not need to calculate the number of threads manually because it is done by ninja by default.